src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTag2.java

```diff
@@ -6,21 +6,31 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
 /**
- * @incomplete This enum needs more code...
+ * Config:
+ * tag = incomplete
+ * tagFormat = .*
+ * severity = ignore
+ * tagSeverity = error
+ * tokens = ANNOTATION_DEF, ENUM_DEF, ANNOTATION_FIELD_DEF, ENUM_CONSTANT_DEF
  */
-enum InputWriteTag2 {
+class InputWriteTag2 {
     /**
-     * @incomplete This enum constant needs more code...
-     */
-    FOO;
-}
+     * @incomplete This enum needs more code...
+     */ // violation
```

Comment:
I have noticed you placed all `// violation` comments not on the same line where expected violation occurred, why so?

Reply:
I forget to commit change on this file. Only this file was left to do changes suggested previously. Done now. Please check.